department-of-veterans-affairs/caseflow
# add cavc dashboard button to post-dispatch actions APPEALS-13221

Resolves [#13221](https://vajira.max.gov/browse/APPEALS-13221?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow&stepId=20)

### Description
- Added CAVC Dashboard button to Post Dispatch Actions based on following conditions:
- If user is from organizations OCC or OAI
- If CAVC decision type is one of the following:
_**other_dismissal**_, **_affirmed_**, or **_settlement_**

### Acceptance Criteria
- [ ] After creating a new CAVC Remand if the following conditions have been met. Then in the Post-dispatch Actions on the Case Details page an "CAVC Dashboard" button is included

1. If the answer to "How are you proceeding?" section on the Add CAVC Remand form is one of the following then the button is shown on return to the Case Details page:
- Affirmed
- Dismissal, other than death
- Settlement


2. If the user is in one of these organizations
- OCC
- OAI

### Testing Plan
1. Log in as CAVC Lit user/admin
2. Search for an appeal that is dispatched.
3. Click on +Add Cavc Remand on the Post Dispatch Actions section at the top.
4. Fill in the form making sure that "How are you proceeding?" section is selected with either of these options

- Dismissal, other than death
- Affirmed
- Settlement

5. Press submit 
6. then switch to an OCC/OAI user and go back to the same case details page.
7. Observe that there is a CAVC Dashboard Button at the top of the page.
8. Click on the CAVC Dashboard Button and observe that you are taken to the cavc dashboard page.
9. Repeat 6-8 with the other OCC/OAI user you did not use.
10. Switch to a non OCC/OAI user and go back to the same case details page
11. Observe that no CAVC Dashboard button appears at the top.

### User Facing Changes

 BEFORE
<img width="1725" alt="before cavc" src="https://user-images.githubusercontent.com/111081469/217665463-47dc74d8-5590-4d54-a163-c08b21404955.png">
AFTER
<img width="1725" alt="after cavc" src="https://user-images.githubusercontent.com/111081469/217665502-4db12e3f-5a0c-48c8-a1cb-45e24be92a47.png">
